operator/zitadel/kinds/iam/zitadel/migration/precontainer_test.go

```diff
@@ -75,3 +76,7 @@ func TestMigration_GrantUserCommand(t *testing.T) {
 	cmd := grantUserCommand(user, file)
 	assert.Equal(t, cmd, equals)
 }
+
+func TestMigration_Image_Default(t *testing.T) {
+
```

Comment:
Empty test?

Reply:
Forgot to delete this.
Tests are [here](https://github.com/caos/zitadel/blob/custom-registry/operator/common/images_test.go)